### PR TITLE
Beautified readme successfully 💅🏼

### PR DESCRIPTION
**Pull request by praxeds**
# Description

Updated the readme.md file to look as friendly and beginner friendly as possible.
I used badges on the header to highlight important dynamic info such as the project's open issues, contributors as well as a link to the Together channel on Discord and a deploy to Cyclic button.

For the design and color palette part, I relied on @Natashka333's mockups alongside @colinborows's logo. All illustrations used on the file are saved under /docs/images and I credited the illustrator under the 'Acknowledgements' section.

I removed the following sections:
- Optimizations
- Lessons Learned
- Next Level
- Next next-level stuff
- The Penultimate Goal
- thinking ahead
- Things to add

Let me know if I wrongly removed any necessary section.

## Type of change

Please select everything applicable. Please, do not delete any lines.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Issue
- [x] Is this related to a specific issue? Is so please specify:
[Readme Improvements #136](https://github.com/Caleb-Cohen/Together/issues/136)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,84 +1,195 @@
...
-Work in Progress
-
-## Optimizations
-
-_(optional)_
-
-Work in Progress
+- ensure `.env` file is located in `server/config` file.
+- `npm install` in the root folder. installs the required dependencies.
+- `npm run build` compiles the build folder. Lint must have no errors for build file command to complete.
```
Comment (Caleb-Cohen) [inline Readme.md:125]: This line can be removed, as this is legacy instructions.

File: Readme.md [hunk 2]
```diff
@@ -1,84 +1,195 @@
...
-**Tech used:** Tailwind, JavaScript, MERN
-
-Work in Progress
-
-## Optimizations
-
-_(optional)_
-
-Work in Progress
+- ensure `.env` file is located in `server/config` file.
```
Comment (Caleb-Cohen) [inline Readme.md:123]: Can you add a link to the .env setup instructions? https://github.com/Caleb-Cohen/Together/wiki/02---Onboarding#development-machine-setup

File: Readme.md [hunk 3]
```diff
@@ -1,84 +1,195 @@
...
 
-## Lessons Learned:
+If you make changes to the front end you must execute `npm run build` otherwise the backend will serve the outdated build file.
 
-Work in Progress
+<br>
+<strong>Still having troubles?</strong>
 
-## Want to Contribute?
+You can join the discussion in the together discord channel <strong>[here](https://discord.com/channels/735923219315425401/1038482732633825442)</strong>.
```
Comment (Caleb-Cohen) [inline Readme.md:133]: Please change to.

"You check check out our [New Developer Onboarding](https://github.com/Caleb-Cohen/Together/wiki/02---Onboarding) in our [wiki.](https://github.com/Caleb-Cohen/Together/wiki) You can also join the discussion in the [together discord channel](https://discord.com/channels/735923219315425401/1038482732633825442) within [100Devs](https://leonnoel.com/100devs/).

File: Readme.md [hunk 4]
```diff
@@ -1,84 +1,195 @@
...
+<table>
+  <tr>
+    <td valign="center">
+     <h3>Resources</h3>
+        · Tailwind Calendar by <a href="https://codepen.io/robstinson/pen/BaKOZry">Rob Stinson</a> @ Codepen
+        <br>
+        · 'Cool Kids' illustrations by <a href="https://blush.design/artists/iDxEJwP2Ha4IrbT6bF88/irene-falgueras">Irene Falgueras</a>
+    </td>
+    <td valign="center" style="width:10%"><img height="120px" src="docs/images/mug.png" alt="Mug"/></td>
+  </tr>
```
Comment (Caleb-Cohen) [inline Readme.md:185]: A bit hard to type up the exact change in the editor, but can you add yourself to this?

"Readme Designed by [Praxeds/Ana](https://github.com/praxeds/)"

File: Readme.md [hunk 5]
```diff
@@ -1,84 +1,195 @@
...
-keep these in mind to prevent the project from being "caged."
-can integrate with slack or Discord and work on other servers besides 100devs
+<br>
 
-Resources:
-https://codepen.io/robstinson/pen/BaKOZry
+If you're interested in contributing, you may:
+* read our [Contributing guide](.github/CONTRIBUTING.md);
+* Visit the [Issues tab](https://github.com/Caleb-Cohen/Together/issues) to request an issue or to open a new issue;
+* You can also convert a "draft todo" to an issue on the [MVP Tasks Project Board.](https://github.com/users/Caleb-Cohen/projects/1/views/1)
```
Comment (Caleb-Cohen) [inline Readme.md:163]: Can we add the join discord instructions here as well?

**Review by Caleb-Cohen — COMMENTED** [review]
Ana, you've completely blown me away with what you accomplished in such a short amount of time. This looks great, and I really appreciate the work that you've put in.

I've added a few minor changes that I would prefer to see, but I'd love to hear your thoughts!

File: Readme.md (hunk 1 above)
Comment (praxeds) [reply Readme.md]: Hey Caleb, I wasn't sure which line were you referring to here so I based myself on the '[Development Machine Setup](https://github.com/Caleb-Cohen/Together/wiki/02---Onboarding#development-machine-setup)' section from the wiki instead for the instructions.

**Comment by praxeds** [comment]
Hi there!
Rewrote some lines according to your review.
Also added a line thanking all the contributors under the Contributions section.

**Pull request merged**